modules/conversion-center/base/widget-contact-button-base.php

```diff
@@ -469,6 +472,73 @@ private function add_style_chat_button_section(): void {
 
 		$this->end_controls_tabs();
 
+		$this->add_control(
+			'style_chat_button_box_divider',
+			[
+				'type' => Controls_Manager::DIVIDER,
+			]
+		);
+
+		$this->add_group_control(
+			Group_Control_Box_Shadow::get_type(),
+			[
+				'name' => 'style_chat_button_box_shadow',
+				'selector' => '{{WRAPPER}} .e-contact-buttons__chat-button',
+			]
+		);
+
+		$this->add_responsive_control(
+			'style_chat_button_animation',
+			[
+				'label' => esc_html__( 'Entrance Animation', 'elementor' ),
+				'type' => Controls_Manager::ANIMATION,
+				'frontend_available' => true,
+			]
+		);
+
+		$this->add_control(
+			'style_chat_button_animation_duration',
+			[
+				'label' => esc_html__( 'Animation Duration', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'default' => [
+					'size' => 800,
+				],
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 3000,
+						'step' => 100,
+					],
+				],
+				'selectors' => [
+					'{{WRAPPER}} .e-contact-buttons' => '--e-contact-button-chat-button-animation-duration: {{SIZE}}ms;',
+				],
+				'condition' => [
+					'style_chat_button_animation!' => '',
+				],
+			]
+		);
```

Comment:
In the Figma, this control is of type SELECT, Though I agree that it is more reasonable for it to be a slider control.
![image](https://github.com/elementor/elementor/assets/94905340/04783155-433e-4136-aff4-df35de97e8e1)

Reply:
I had to deviate from the designs here because having "slow/normal/fast" didn't really serve the purpose for the animation duration. I'm discussing this with Aviva to see if we can keep the slider or change it to the original select.